development/tools/jobs/control-plane/components_test.go

```diff
@@ -0,0 +1,89 @@
+package control_plane
+
+import (
+	"github.com/kyma-project/test-infra/development/tools/jobs/releases"
+	"github.com/kyma-project/test-infra/development/tools/jobs/tester"
+	"github.com/kyma-project/test-infra/development/tools/jobs/tester/jobsuite"
+	"path"
+	"testing"
+)
+
+const (
+	jobBasePath = "./../../../../prow/jobs"
+)
+
+var components = []struct {
+	name              string
+	image             string
+	suite             func(config *jobsuite.Config) jobsuite.Suite
+	additionalOptions []jobsuite.Option
+}{
+	{
+		name:  "provisioner",
+		image: tester.ImageBootstrap20181204,
+		suite: tester.NewGenericComponentSuite,
+		additionalOptions: []jobsuite.Option{
+			jobsuite.JobFileSuffix("control-plane-generic"),
```

Comment:
Had to change the suffix due to an error:
```"error":"duplicated basename is not allowed: provisioner-generic.yaml"```
After removing components from the Compass repo, the suffix can be removed

Reply:
I suggest using `kcp` suffix in the meantime 🙂